docs/wiki/configurator/vtx-tab.md

```diff
@@ -25,7 +25,7 @@ It is important to have VTX Telemetry (SmartAudio or Tramp) enabled in the Ports
 | **Power**                    | Select the power level to use.                                                                                                                                                                                                                                                                                                                                                      |
 | **Pit Mode**                 | When enabled, the VTX enters in a very low power mode to let the quad be on at the bench without disturbing other pilots. Usually the range of this mode is less than 5m.<br/>NOTE: Some protocols, like SmartAudio, can't enable Pit Mode via software after power-up.                                                                                                             |
 | **Pit Mode frequency**       | Select the frequency to use in Pit Mode.                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Maybe it sounds obvious but thinking to add a "Note" here too to say that 0 means to stay in the current frequency

Reply:
i'm unsure that this is correct.  looking at some old code (BF 3.5, 4.0), pitmode 0 defined as setting "out-of-range"; however, maybe this is true for modern VTX tables. i am unsure.